chapters/ch06/3.spell-correction.ipynb

```diff
@@ -28,7 +28,7 @@
     "import json\n",
     "import numpy\n",
     "import pandas\n",
-    "from aips import get_engine\n",
+    "from aips import get_engine, set_engine\n",
```

Comment:
is `set_engine` used?

Reply:
I'll remove all set_engine's from the codebase.